plugins/mcreator-localization/lang/texts.properties

```diff
@@ -1405,6 +1405,22 @@ elementgui.potion.error_potion_needs_icon=Potion needs to have icon
 elementgui.potion.error_potion_needs_display_name=Potion needs a display name
 elementgui.potion.error_potion_effect_needs_display_name=Potion effect needs a display name
 elementgui.potion.enable_bottles=Enable bottles
+elementgui.procedure.dependencies_added=<html><div style='width: 110px; padding: 2px; background: #444444;'><span style='background: yellow; color: black; font-wight: bold;'>&nbsp;&nbsp;!&nbsp;&nbsp;</span>&nbsp;New dependencies were added. Make sure to use the ones that the trigger you use provides.
+elementgui.procedure.dependencies_not_provided=<html><div style='width: 110px; padding: 2px; background: #444444;'><span style='background: red; color: white; font-wight: bold;'>&nbsp;X&nbsp;</span>&nbsp;You have selected external trigger that does not provide the following dependencies: {0}
+elementgui.procedure.cancelable_trigger=<html><div style='width: 100px; font-size: 9px;'>Cancelable
+elementgui.procedure.can_specify_result_trigger=<html><div style='width: 100px; font-size: 9px;'>Can specify result
+elementgui.procedure.global_trigger_unsupported=Selected global trigger is not supported by the selected generator. It will be ignored.
+elementgui.procedure.global_trigger_not_activated=Selected global trigger requires {0} enabled in workspace settings, or the current generator does not support it
+elementgui.procedure.global_trigger_does_not_exist=Your procedure uses a global trigger that does not exist
+elementgui.procedure.return_type=Return type
+elementgui.procedure.local_variables=Local variables
+elementgui.procedure.name_already_exists=This name already exists
+elementgui.procedure.confirm_delete_var_msg=<html>Are you sure that you want to delete this variable?<br>If this variable is in use, this action might cause compilation errors.
+elementgui.procedure.confirm_delete_var_title=Confirmation
+elementgui.procedure.required_dependencies=Required dependencies
+elementgui.procedure.provided_dependencies=  Provided dependencies
+elementgui.procedure.external_trigger_does_not_provide_all_dependencies=External trigger you selected does not provide<br>all the dependencies your procedure requires!
+

```

Comment:
Is empty line (1423) needed?

Reply:
Fixed. Please consider this PR is still draft, the point of this is to avoid unnecessary reviews.